safe/gui/tools/wizard/wizard_dialog.py

```diff
@@ -876,3 +880,29 @@ def save_current_keywords(self):
                     current_keywords['inasafe_default_values'].items()):
                 set_inasafe_default_value_qsetting(
                     self.setting, RECENT, key, value)
+
+    # Adapted from https://stackoverflow.com/a/43126946/1198772
+    def resizeEvent(self, event):
+        """Emit custom signal when the window is re-sized.
+
+        :param event: The re-sized event.
+        :type event: QResizeEvent
+        """
+        self.resized.emit()
+        return super(WizardDialog, self).resizeEvent(event)
+
+    def after_resize(self):
+        """Method after resizing the window."""
+        if self.get_current_step() == self.step_kw_fields_mapping:
+            # Use MinimumExpanding when on the Field Mapping Step
```

Comment:
Hum, why only this page? Can't it be extended to all other pages? cf the ticket I put

Reply:
Ah yes. Thanks. I have made it applied to all step. Turns out, the most important one is set the max height.